llvm/lib/ObjCopy/ELF/ELFObject.cpp

```diff
@@ -2282,8 +2291,12 @@ Error Object::replaceSections(
 Error Object::removeSymbols(function_ref<bool(const Symbol &)> ToRemove) {
   if (SymbolTable)
     for (const SecPtr &Sec : Sections)
-      if (Error E = Sec->removeSymbols(ToRemove))
+      if (Error E = Sec->removeSymbols(ToRemove)){
```

Comment:
I think you need to run your changes through clang-format.

```suggestion
      if (Error E = Sec->removeSymbols(ToRemove)) {
```